assets/src/components/map/markers/locationMarker.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import { ReactMarker } from "../utilities/reactMarker"
+import { LocationSearchResult } from "../../../models/locationSearchResult"
+import { LocationDotIcon } from "../../../helpers/icon"
+import Leaflet from "leaflet"
+
+export const LocationMarkerIcon = ({ selected }: { selected?: boolean }) => {
+  return (
+    <LocationDotIcon
+      className={
+        "c-location-dot-icon" +
+        (selected ? " c-location-dot-icon--selected" : "")
+      }
+    />
+  )
+}
+
+export const LocationMarker = ({
```

Comment:
**suggestion (non-blocking):** now that we also have the `<UserLocationMarker/>` component I've sort of wanted to come up with a more descriptive name for this to differentiate it (maybe something indicating that it's a geocoded location from a search result?), but just separating it out into its own module is already an improvement on its own.

Reply:
`<LocationSearchMarker />`?

Reply:
@joshlarson I like `<LocationSearchMarker/>`, were you going to make that change?

Reply:
Thanks for reminding me though - it had completely slipped my mind.

Reply:
@lemald Done!